consensus/consensus_v2.go

```diff
@@ -157,7 +157,6 @@ func (consensus *Consensus) finalCommit() {
 		consensus.getLogger().Err(err).
 			Uint64("beforeCatchupBlockNum", beforeCatchupNum).
 			Msg("[finalCommit] Leader failed to commit the confirmed block")
-		return
```

Comment:
what's the reason to remove the return here?

Reply:
just in case something weird happened in the leader logic, it should still try propose new blocks. As long as validators still verify the blocks and messages. It doesn't hurt to try move forward optimisitically.